Headers/Foundation/NSString.h

```diff
@@ -208,10 +221,42 @@ typedef enum _NSStringEncoding
   NSISOLatin8StringEncoding = 62,	// ISO-8859-14
   NSISOLatin9StringEncoding = 63,	// ISO-8859-15; Replaces ISOLatin1
   NSGB2312StringEncoding = 56,
-  NSUTF7StringEncoding = 64,		// RFC 2152
-  NSGSM0338StringEncoding,		// GSM (mobile phone) default alphabet
-  NSBIG5StringEncoding,			// Traditional chinese
-  NSKoreanEUCStringEncoding		// Korean
+  NSUTF7StringEncoding = 64, // RFC 2152
+  NSGSM0338StringEncoding = 65, // GSM (mobile phone) default alphabet
+  NSBIG5StringEncoding = 66, // Traditional chinese
+  NSKoreanEUCStringEncoding = 67, // Korean
+
+  NSDOSLatinUSStringEncoding = 68, // code page 437
+  NSDOSGreekStringEncoding, // code page 737 (formerly code page 437G) */
+  NSDOSBalticRimStringEncoding, // code page 775
+  NSDOSLatin1StringEncoding, // code page 850, "Multilingual"
+  NSDOSGreek1StringEncoding, // code page 851
+  NSDOSLatin2StringEncoding, // code page 852, Slavic
+  NSDOSCyrillicStringEncoding, // code page 855, IBM Cyrillic
+  NSDOSTurkishStringEncoding, // code page 857, IBM Turkish
+  NSDOICortugueseStringEncoding, // code page 860
+  NSDOSIcelandicStringEncoding, // code page 861
+  NSDOSHebrewStringEncoding, // code page 862
+  NSDOSCanadianFrenchStringEncoding, // code page 863
+  NSDOSArabicStringEncoding, // code page 864
+  NSDOSNordicStringEncoding, // code page 865
+  NSDOSRussianStringEncoding, // code page 866
+  NSDOSGreek2StringEncoding, // code page 869, IBM Modern Greek
+  NSDOSThaiStringEncoding, // code page 874, also for Windows
+  NSDOSJapaneseStringEncoding, // code page 932, also for Windows
+  NSDOSChineseSimplifStringEncoding, // code page 936, also for Windows
+  NSDOSKoreanStringEncoding, // code page 949, also for Windows; Unified Hangul Code
+  NSDOSChineseTradStringEncoding, // code page 950, also for Windows
+  // NSWindowsLatin1StringEncoding, // code page 1252, NSWindowsCP1252StringEncoding
+  // NSWindowsLatin2StringEncoding, // code page 1250, Central Europe, NSWindowsCP1250StringEncoding
+  // NSWindowsCyrillicStringEncoding, // code page 1251, Slavic Cyrillic. NSWindowsCP1251StringEncoding
+  // NSWindowsGreekStringEncoding, // code page 1253, NSWindowsCP1253StringEncoding
+  // NSWindowsLatin5StringEncoding, // code page 1254, Turkish, NSWindowsCP1254StringEncoding
+  NSWindowsHebrewStringEncoding, // code page 1255
```

Comment:
If we leave out some definitions aren't the following a bit off?  Should we explicitly specify the number here?

Reply:
do you suggest that we should set number explicitly for all cases? I'm in favor of that.

for these cases that are commented out, i think i could change it to `NSWindowsLatin1StringEncoding = NSWindowsCP1252StringEncoding`